packages/zoe/src/zoeService/zoe.js

```diff
@@ -431,11 +438,17 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
             );
 
             const offerResultPromiseKit = makePromiseKit();
-            // Don't trigger Node.js's UnhandledPromiseRejectionWarning
-            offerResultPromiseKit.promise.catch(_ => {});
+            offerResultPromiseKit.promise.catch(err => {
+              console.log('An offerHandler has errored.');
+              console.error(err);
+              // Error suppressed to not trigger Node.js's UnhandledPromiseRejectionWarning
```

Comment:
Why is this one different?

Reply:
Because it's the offerResult promise - we only want the offerResult promise to reject, not for the rejected promise to cause the offer to throw

Reply:
I was thinking of asking for a test, but decided to check for myself. The brokenContract tests show this is working correctly.

Reply:
> Because it's the offerResult promise - we only want the offerResult promise to reject, not for the rejected promise to cause the offer to throw

The offer won't throw even if you don't suppress... it will just trigger the `UnhandledPromiseRejection` callback.  The reason we don't want that is because we expect the caller to examine the offerResult promise and surface it as needed by them.

No need to log here, based on that assumption (that the caller will examine the promise).